jekyll/_plugins/uid-permalinks.rb

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Autodesk, Inc.  All rights reserved.
+#
+# Use of this software is subject to the terms of the Autodesk license agreement
+# provided at the time of installation or download, or which otherwise accompanies
+# this software in either electronic or hard copy form.
+#
+
+# Error raised when two page UIDs collide, meaning two different pages will
+# have the same URL.
```

Comment:
when does this happen, if things are auto generated? might be worth adding a comment about?

Reply:
Added to the comment.